src/ngAnimate/shared.js

```diff
@@ -69,10 +55,6 @@ var ANIMATION_DURATION_PROP = ANIMATION_PROP + DURATION_KEY;
 var TRANSITION_DELAY_PROP = TRANSITION_PROP + DELAY_KEY;
 var TRANSITION_DURATION_PROP = TRANSITION_PROP + DURATION_KEY;
 
-var isPromiseLike = function(p) {
-  return p && p.then ? true : false;
-};
-
```

Comment:
I checked all global entries from `.jshistrc` to see in which files they are used and if they are still needed. I removed a few that weren't used any more.
This is only used in test (maybe it used to be used here as well), so I moved it there.